packages/logic-utils/api.d.ts

```diff
@@ -128,3 +129,48 @@ export function getUI5NodeFromXMLElementNamespace(
   isDefault: boolean;
   isXmlnsDefined: boolean;
 };
+
+/**
+ * Get the deprecated message. The returned string contains jsdoc tags.
+ *
+ * @param opts.title - The opening sentance for the deprecation message. It should not contain a dot. "Deprecated" by default.
+ */
+export function getDeprecationMessage(opts: {
+  title?: string;
+  since: string | undefined;
+  text: string | undefined;
+}): string;
+
+/**
+ * Get a snippet (first line) of the deprecated documentation without jsdoc tags.
+ *
+ * @param title - The opening sentance for the deprecation message. It should not contain a dot. "Deprecated" by default.
+ * @param deprecatedInfo
+ * @param model
+ */
+export function getDeprecationPlainTextSnippet(
+  title: string | undefined,
+  deprecatedInfo: UI5DeprecatedInfo,
+  model: UI5SemanticModel
+): string;
+
+/**
+ * Convert jsdoc description to markdown format string
+ * @param jsdocDescription
+ * @param model
+ */
+export function convertJSDocToMarkdown(
```

Comment:
completely subjective:
- config object pattern vs ordered args.
- normally over kill for 2 args, but maybe not when it is a public api to avoid future breaking changes?

Your choice.